Source/WebCore/rendering/ImageQualityController.cpp

```diff
@@ -117,6 +117,18 @@ std::optional<InterpolationQuality> ImageQualityController::interpolationQuality
     return std::nullopt;
 }
 
+InterpolationQuality ImageQualityController::chooseInterpolationQuality(GraphicsContext& context, RenderObject* object, Image& image)
```

Comment:
We should name this in a way that makes it clear whether callers should use this one or the existing one, so maybe `chooseInterpolationQualityForSVG` or something.

I feel like we probably do want to have the kind of interpolation quality heuristics that the existing function has, though I imagine with its size calculations it doesn't work well with SVG render trees when determining the area of the layer that the image covers.